sdk/identity/azure-identity/src/azure_cli_credential.cpp

```diff
@@ -0,0 +1,654 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/identity/azure_cli_credential.hpp"
+
```

Comment:
Why separate sorting groups here (clang-format treats each <LF> delimited block as a separate sorting group)? Is there a reason you don't want to group azure_cli_credential.hpp with token_credential_impl.hpp?

Reply:
I do it this way because first comes the header file to which this file corresponds. No bigger meaning here, other than readability.